File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/dashboard/data/UserData.java
package org.wikiedufoundation.wikiedudashboard.dashboard.data;

public class UserData {
    private String username;

    public UserData(String username) {
        this.username = username;
    }

    public String getUsername() {
        return username;
    }

    @Override
    public String toString() {
        return "UserData{" +
                "username='" + username + '\'' +
                '}';
    }
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/dashboard/data/UserData.kt
package org.wikiedufoundation.wikiedudashboard.ui.dashboard.data

import com.google.gson.annotations.SerializedName

/**
 * [UserData] model class
 * @constructor primary constructor to initialize properties and variables
 * ***/
data class UserData(
    @SerializedName("username")
    val userName: String
)
